lambda-views/models/thought_experiment_only/page_views.sql

```diff
@@ -0,0 +1,39 @@
+{{
```

Comment:
Since these would cause errors if we `dbt run`, maybe we should put them at:
```
lambda-views/thought-experiment-models/page_views.sql
```

Well, then again, no one can run these anyway unless they have access to our Snowflake account

Reply:
I just disabled them in `dbt_project.yml`, but happy to put them wherever they can be most edifying + least confusing